lib/semantics/check-select-stmt.cc

```diff
@@ -0,0 +1,112 @@
+//==check-select-stmt.cc - Checker for select-case, select-rank
```

Comment:
The comment says you're checking `SELECT CASE` and `SELECT RANK` but this file only contains code for `SELECT RANK`.

There's a [pull request](https://github.com/flang-compiler/f18/pull/889) outstanding for `SELECT CASE` that uses the same file name `check-select-stmt.cc`; are you two coordinating your work?

Reply:
Yes :)